dams-key-server/src/error.rs

```diff
@@ -1,3 +1,12 @@
+use crate::database::log as Log;
```

Comment:
```suggestion
use crate::database::log;
```

This is more of a Javascript pattern. We shouldn't use it in Rust code.

Reply:
Okay! I started this for `user/User` so there wouldn't be a name conflict with variables, and I kept the pattern for this too. Would changing it to just `log` conflict with the `log()` function though?

Reply:
I don't think it would. I believe the compiler will consider the context before reporting it as a conflict.

If we do end up with conflicts, we can do something like `use crate::database::log as logging` or just rename the module. I just want to make sure we stick to Rust's naming conventions.